compiler/testData/codegen/bytecodeText/companion/inlineFunctionCompanionPropertyAccess.kt

```diff
@@ -0,0 +1,45 @@
+class A {
+    companion object {
+        val s = "OK"
+        var v = "NOT OK"
+    }
+
+    inline fun f(): String = s
+
+    inline fun g() {
+        v = "OK"
+    }
+}
+
+// One direct `A.s` access in the accessibility bridge `access$getS$cp`.
+// 1 GETSTATIC A.s
+
+// One direct `A.v` set in the accessibility bridge `access$setV$cp`.
+// One direct `A.v` set in `A.<clinit>`
+// 2 PUTSTATIC A.v
+
+// JVM_TEMPLATES:
+
+// One call to the getter/setter on the companion object from f and one from g.
+// 1 INVOKEVIRTUAL A\$Companion.getS
+// 1 INVOKEVIRTUAL A\$Companion.setV
+
+// One call to the accessibility bridge `access$setV$cp` from Companion.setV.
+// 1 INVOKESTATIC A.access\$setV\$cp
+
+// One call to the accessibility bridge `access$getS$cp` from Companion.getS.
+// 1 INVOKESTATIC A.access\$getS\$cp
+
+// JVM_IR_TEMPLATES:
+
+// No calls to the getter/setter on the companion object.
+// 0 INVOKEVIRTUAL A\$Companion.getS
```

Comment:
>So instead of calling Companion.getS -> A.access$getS$cp we directly call A.access$getS$cp which looks fine to me?

I'm expect that next change wouldn't be binary compatible cause of synthetic accessor usage:
```
class A {
    companion object {
        val s: String
            get() = "Ok"
        var v : String 
            get() = "NOT OK"
            set(value) {}
    }

    inline fun f(): String = s

    inline fun g() {
        v = "OK"
    }
}
```

Reply:
These are not `DEFAULT_PROPERTY_ACCESSORS` so they are filtered out and `A$Companion.setV` and `A$Companion.getS` will be used from `g` and `f`, so this should generate the same code as before. Let me add a bytecode text test checking that as well. :)

Reply:
I meant that if original code 
```
class A {
    companion object {
        val s = "OK"
        var v = "NOT OK"
    }

    inline fun f(): String = s

    inline fun g() {
        v = "OK"
    }
}
```
would be changed to example above it would break binary compatibility

Reply:
Oh my, I'm sorry @max-kammerer I was moving way too fast here. I should have understood what you meant. Thank you for catching that! :-) 

I filtered out inline functions as well, so that we only use direct access for code that actually stays on the class of the property backing field.